__tests__/__fixtures__/output/node/fetch/application-json.js

```diff
@@ -4,7 +4,14 @@ const url = 'https://httpbin.org/anything';
 const options = {
   method: 'POST',
   headers: {'content-type': 'application/json'},
-  body: '{"number":1,"string":"f\"oo","arr":[1,2,3],"nested":{"a":"b"},"arr_mix":[1,"a",{"arr_mix_nested":{}}],"boolean":false}'
```

Comment:
Prior to this fix, see `json` in this returned payload:

```js
{
  args: {},
  data: '{"number":1,"string":"f"oo","arr":[1,2,3],"nested":{"a":"b"},"arr_mix":[1,"a",{"arr_mix_nested":{}}],"boolean":false}',
  files: {},
  form: {},
  headers: {
    Accept: '*/*',
    'Accept-Encoding': 'gzip,deflate',
    'Content-Length': '117',
    'Content-Type': 'application/json',
    Host: 'httpbin.org',
    'User-Agent': 'node-fetch/1.0 (+https://github.com/bitinn/node-fetch)',
    'X-Amzn-Trace-Id': 'Root=1-613fc993-7f43a7fc522f744c69c69531'
  },
  json: null,
  method: 'POST',
  origin: 'REDACTED',
  url: 'https://httpbin.org/anything'
}
```

And now:

```js
{
  args: {},
  data: '{"number":1,"string":"f\\"oo","arr":[1,2,3],"nested":{"a":"b"},"arr_mix":[1,"a",{"arr_mix_nested":{}}],"boolean":false}',
  files: {},
  form: {},
  headers: {
    Accept: '*/*',
    'Accept-Encoding': 'gzip,deflate',
    'Content-Length': '118',
    'Content-Type': 'application/json',
    Host: 'httpbin.org',
    'User-Agent': 'node-fetch/1.0 (+https://github.com/bitinn/node-fetch)',
    'X-Amzn-Trace-Id': 'Root=1-613fc98a-181e57cc5b875d915032bfc7'
  },
  json: {
    arr: [ 1, 2, 3 ],
    arr_mix: [ 1, 'a', [Object] ],
    boolean: false,
    nested: { a: 'b' },
    number: 1,
    string: 'f"oo'
  },
  method: 'POST',
  origin: '98.42.248.146',
  url: 'https://httpbin.org/anything'
}
```

Reply:
Not sure what you mean by `useObjectBody`, but it looks like the original  test had invalid JSON. the `"f\"` should have been `"f\\"'